.github/workflows/ci.yml

```
@@ -559,17 +559,17 @@ jobs:
         run: |
           python3 stuff/db-migrate.py \
             --skip-container-check \
-            migrate --databases=omegaup-test
+            migrate --databases=omegaup
 
       - name: Validate database schema
         run: |
           python3 stuff/policy-tool.py \
             --skip-container-check \
-            --database=omegaup-test \
+            --database=omegaup \
             validate
           python3 stuff/database_schema.py \
             --skip-container-check \
-            --database=omegaup-test \
+            --database=omegaup \
             validate --all < /dev/null
```

faltaría inicializar composer

```suggestion
            validate --all < /dev/null
            
      - name: Install composer dependencies
        run: |
          docker-compose exec -T frontend composer install --prefer-dist --no-progress
```